### ISSUE TITLE
Stop action from 60-day auto-disable

### ISSUE THREAD

**Issue by mpadge** [ropensci-org/badges#7]
This repo generally doesn't need any modifications, but GitHub now disables actions on 60 days of inactivity, which means the badge server automatically stops updating, leading to things like https://github.com/ropensci/software-review/issues/486#issuecomment-1015792134. We definitely need a way to prevent this particular workflow from being auto-disabled. @jeroen Any brilliant suggestions here?

**Comment by jeroen** [comment]
I think the only solution is to automatically push something regularly into the repo, using a token that is not the standard one from the actual, but an actual PAT from somebody.

We could also consider switching to r-universe badges: https://ropensci.r-universe.dev/ui#badges

**Comment by mpadge** [comment]
> I think the only solution is to automatically push something regularly into the repo, using a token that is not the standard one from the actual, but an actual PAT from somebody.

Yeah, that was my thought too, but can't do it on GitHub without running the risk of cascading auto-disables. Only really safe way would be to run that from a truly external service, which then gets a bit messy. r-universe badges might be a good solution, but these badges are quite different because they need to be aligned with the registry, especially for stats peer review - i'll schedule that in for a staff/standup chat time soon-ish. Thanks!

**Comment by mpadge** [comment]
@jeroen Do you know what the status is of GitHub auto-disable? This service has now been running without break for months and months. Do you know of any changes to policy?
